src/edges_estimate/foregrounds.py

```diff
@@ -302,3 +302,44 @@ def calculate(self, ctx, **params):
                     break  # only add to one thing, otherwise it's doubling up.
                 else:
                     ctx[key] *= bias
+
+
+@attr.s
+class LogPoly(Foreground):
+    """
+    LogPoly model from Sims et.al. 2020 (Equation 18)
+    T_{Fg} = 10^sum(d_i*log10(ν/ν_0)^i)_{i=1 to N}
```

Comment:
why is there no `i=0` term? wouldn't that give the basic power-law behavior?

Reply:
This is a good question. Maybe @PSims can confirm the equation? I followed the equation in the paper.

Reply:
That's a good spot, i=0 should be in there (it sets the foreground temperature at the reference frequency). It's listed correctly in Tab. 2.